flit.ini

```diff
@@ -0,0 +1,16 @@
+[metadata]
+module = nbgrader
+author = Jessica Hamrick
```

Comment:
Thanks for putting me as the author :-) But I think maybe it would be better to do "Jupyter Development Team" since that's what all the other Jupyter projects (like JupyterHub) have.